pages/standards-guidelines/wcag/non-web-ict.md

```diff
@@ -44,24 +44,64 @@ Quick link:
+
 *Background:* **WCAG 2.0** is a normative web standard — it is a W3C Recommendation and an ISO International Standard ([ISO/IEC 40500:2012](https://www.w3.org/QA/2012/10/wcag_20_is_now_also_isoiec_405.html)) — that explains how to make web content (including static web pages, dynamic web applications, etc.) more accessible to people with disabilities. WCAG 2.1 was published in 2018 and **WCAG 2.2** was published in 2023. The [WCAG Overview](/standards-guidelines/wcag/) introduces the WCAG documents and links to supporting resources such as [[WCAG 2.0 at a Glance]](/standards-guidelines/wcag/20/glance/) and [How to Meet WCAG 2 (Quick Reference)](https://www.w3.org/WAI/WCAG20/quickref/).
 
-*This document:* **WCAG2ICT** is a W3C Group Note that is informative — it is not normative and does not set requirements. It provides guidance on the application of WCAG 2 principles, guidelines, and success criteria to non-web documents and software. WCAG2ICT was developed to provide an authoritative interpretation of how WCAG 2 can apply in different contexts. For information on the scope of WCAG2ICT, see the [Introduction section of WCAG2ICT](https://www.w3.org/TR/wcag2ict/#intro).
+### Goals
+The objective of the WCAG2ICT Task Force is to develop documentation describing how WCAG 2.x and its principles, guidelines, and success criteria could apply to non-Web Information and Communications Technologies (ICT).
+
+### Intent and Usage of WCAG2ICT
+
+WCAG2ICT, like any other  [W3C Group Note](https://www.w3.org/policies/process/#Note), is informative — it is not normative and does not set requirements. It provides guidance on the application of WCAG 2 principles, guidelines, and success criteria to non-web documents and software. The WCAG2ICT Group Note was developed to provide an authoritative interpretation of how WCAG 2 can apply in different contexts.
+
+The main focus of the WCAG2ICT Group Note is to provide verbiage replacements for web-based language in the WCAG success criteria and definitions as well as provide additional insights into where WCAG criteria may make assumptions of the presence and functionality of a user agent and assistive technology. This is because not all software has a user agent or platform software with accessibility APIs and services, nor do all technologies have or support assistive technology that acts on programmatic information. 
+
+While the WCAG2ICT Group Note provides guidance on applying WCAG in non-web contexts, at this time it may not provide sufficient clarity for some success criteria on how they can be applied to electronic documents whose source is driven by non-web markup technologies when they are delivered via a website. Likewise, there may not be sufficient clarity for mobile applications.  
```

Comment:
Put this on a more positive way

```suggestion
While the WCAG2ICT Group Note provides guidance on applying WCAG in non-web contexts, application of some success criteria to electronic documents whose source is driven by non-web markup technologies when they are delivered via a website may benefit from more specific guidance. Likewise, guidance for mobile applications may also benefit from a greater level of detail.  
```

Reply:
Accepted as suggestion per Task Force review.